install/environment.yml

```diff
@@ -6,3 +6,4 @@ dependencies:
   - python>=3.10, <3.12
   - snakemake<8
   - graphviz
+  - pyopenssl>=23.2.0
```

Comment:
The tests were failing with https://github.com/conda/conda/issues/13619
I had to install pyopenssl